docker-compose.yml

```
@@ -84,7 +91,7 @@ services:
     container_name: phpmyadmin
     environment:
       PMA_HOST: db
-      PMA_PORT: 3306
+      PMA_PORT: 3312
```

Since it's referencing the internal name, the internal port should be used instead:

```suggestion
      PMA_PORT: 3306
```